JASP-Engine/JASP/R/regressionloglinearbayesian.R

```diff
@@ -15,811 +15,576 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-RegressionLogLinearBayesian <- function(dataset, options, perform="run", callback, ...) {
-    # Init end result container
-    endResults <- list()
-    
-    # subsresult containter 
-    results <- list()
-    meta <- list()
-    .meta <-  list(list(name="title", type="title"),
-                   list(name="table", type="table"),
-                   list(name="posteriorTable", type="table"),
-                   list(name="Bayesianlogregression", type="table"),
-                   list(name="BayesianSublogregression", type="table")
-    )
-    
-    
-    results[[".meta"]] <- .meta
-    results[["title"]] <- "Bayesian Log-Linear Regression"
-    
-    logLinearBayesianCitations <- 	list(
-        "Overstall, A., & King, R. (2014). conting: an R package for Bayesian analysis of complete and incomplete contingency tables. Journal of Statistical Software, 58(7), 1-27."
-    )
-    
-    # Init table
-    posteriorTable <- list()
-    
-    posteriorTable[["title"]] <- "Model Comparison"
-    posteriorTable[["citation"]] <- logLinearBayesianCitations
+RegressionLogLinearBayesian <- function(jaspResults, dataset = NULL, options, ...) {
+  ready <- length(options$factors) != 0
```

Comment:
```suggestion
  ready <- length(options$factors) > 1
```

Reply:
This ensures you don't get an error if you have 1 factor and tick `estimates` or `Submodel`